changelog/v0.10.24/refresh-operator-cache.yaml

```diff
@@ -0,0 +1,5 @@
+changelog:
+  - type: FIX
+    description: Allow updates to a KubeInstaller cache
+    issueLink: https://github.com/solo-io/sm-marketplace/issues/802
```

Comment:
should link to an issue from a public repo so the link will be accessible to non-solo people